### DISCUSSION TITLE
blog/2023/03/21/aragorn-dunedan-numenorean-simulation/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#79] [Blog comments]
# blog/2023/03/21/aragorn-dunedan-numenorean-simulation/

Use statistical simulation and a hidden table of Númenórean ages from Tolkien’s unpublished works to convert Aragorn’s Dúnedan years to actual human years

https://www.andrewheiss.com/blog/2023/03/21/aragorn-dunedan-numenorean-simulation/

**Comment by etiennebacher** [comment]
That's probably the best summary of the Ages in Tolkien's world I have read, thank you! (probably wasn't the objective of this post though 😄 )

**Comment by agnx93** [comment]
Was re-watching the trilogy tonight and thought: "Aragorn looks really young here, how old is he really?"

Then, here I am. Very good post, thank you for your time, very fun read too <3
